src/file_saver.rs

```diff
@@ -1,10 +1,10 @@
-use std::fs;
-use std::fs::File;
-use std::io::Result;
+use std::fs::{self, File};
+use std::io::{Result, Write};
 use std::path::Path;
+use writer::untabber::Untabber;
 
 pub fn save_to_file<P, F>(path: P, make_backup: bool, closure: &mut F) where
```

Comment:
Ah yes, a popular confusion point. The signature is not just signature, but also a pattern match. You can put `mut closure: F` there. Or rebind in the body:
```
let mut closure = closure;
```